packages/ui/src/helpers/authenticator/getRoute.ts

```diff
@@ -39,18 +41,14 @@ export const getRoute = (
     case actorState?.matches('confirmVerifyUserAttribute'):
       return 'confirmVerifyUser';
     case state.matches('getCurrentUser'):
-    case actorState.matches('fetchUserAttributes'):
+    case actorState?.matches('fetchUserAttributes'):
```

Comment:
Why is the optional chaining now required here?

Reply:
Didn't cause any issues for me but each of the other cases for actorState use optional chaining so assumed it was a mistake